cypress/e2e/login/login.spec.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('Login', () => {
```

Comment:
@AmnaShahid96 the phone number and password should be saved in environment variables and being passed to this ticket instead of hardcoding them in the same line.